tests/browserstack-remote.js

```diff
@@ -0,0 +1,87 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Learn more about configuring this file at <https://github.com/theintern/intern/wiki/Configuring-Intern>.
+// These default settings work OK for most people. The options that *must* be changed below are the
+// packages, suites, excludeInstrumentation, and (if you want functional tests) functionalSuites.
+define(["intern"], function(intern, topic) {
+  "use strict";
+  var args = intern.args;
+  var siteRoot = args.siteRoot ? args.siteRoot : "http://localhost:5000";
+
+  if (topic) {
+    topic.subscribe("/suite/start", function(suite) {
+      /* eslint-disable no-console*/
+      console.log("Running: " + suite.name);
+    });
+  }
+
+  return {
+    // Configuration object for webcompat
+    wc: {
+      pageLoadTimeout: args.wcPageLoadTimeout
+        ? parseInt(args.wcPageLoadTimeout, 10)
+        : 10000
+    },
+
+    // The port on which the instrumenting proxy will listen
+    proxyPort: 9090,
+
+    // A fully qualified URL to the Intern proxy
+    proxyUrl: "http://127.0.0.1:9090/",
+    siteRoot: siteRoot,
+
+    capabilities: {
+      //"browserstack.local": false for running tests on travis
+      "browserstack.local": true,
+      "browserstack.video": true,
+      fixSessionCapabilities: true,
+      remoteFiles: true
+    },
+
+    // Required for BrowserStack, maximum number of simultaneous integration tests allowed
+    maxConcurrency: 2,
+
+    environments: [
+      {
+        browser: "firefox",
+        browser_version: "54",
```

Comment:
Do they have any notion of "latest", or does this need to be updated periodically?